test/test_lazy_import_errors.py

```diff
@@ -14,7 +14,7 @@
 
 
 ######################## Tests for Direct Invocation of the Context Manager #######################
-def test_lazy_import_sad_package(reset_sys_modules):
+def test_lazy_import_sad_package():
```

Comment:
After the change, `reset_sys_modules` import is unused in this file

Reply:
So this is actually annoying, but you still need to import it so that the `autouse=True` is picked up, otherwise it doesn't get used.